app/scripts/models/account.js

```diff
@@ -360,11 +358,7 @@ define([
           .then(function (client) {
             profileClient = client;
             var accessToken = self.get('accessToken');
-            if (accessToken) {
```

Comment:
Yep, I refactored so that `self.profileClient()` always results in an  error or an `accessToken`.